src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java

```diff
@@ -210,11 +210,25 @@ public boolean siVisualizzaUnaDelegaConNomeDelegato(String nome, String cognome)
     }
 
     public String vericaStatoDelega() {
-        By statoDelegaBy = By.xpath("//div[@id='chip-status-success']/span");
-        this.getWebDriverWait(30).withMessage("Non si trova nessuno stato delega").until(ExpectedConditions.visibilityOfElementLocated(statoDelegaBy));
+        By statoDelegaBy = By.id("chip-status-success");
+        getWebDriverWait(30).withMessage("Non si trova nessuno stato delega").until(ExpectedConditions.visibilityOfElementLocated(statoDelegaBy));
         WebElement statoDelega = this.element(statoDelegaBy);
         return statoDelega.getText();
     }
 
 
+    public void checkModaleMostraCodice() {
+        try {
+            By titoloModale = By.id("dialog-title");
+            By sottotitoloModale = By.id("dialog-description");
+            By codiceVerifica = By.xpath("//div[@data-testid='codeInput(0)']");
```

Comment:
You're taking only the first one, please verify all 5

Reply:
done